build.sbt

```diff
@@ -21,6 +21,7 @@ val coreDependencies = Seq(
   "org.slf4j" % "log4j-over-slf4j" % "1.7.12",
   "org.scalatest" %% "scalatest" % "2.2.4" % "test",
   "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2",
+  "org.scalaj" %% "scalaj-collection" % "1.6",
```

Comment:
According to the documentation these converters provide the following two-way conversion:
`scala.collection.mutable.Map <=> java.util.Map` but what is needed is at least this one-way conversion: `java.util.Map => scala.collection.Map`, and this does not exist in the std library.